src/components/forms/JurisdictionSelect/index.tsx

```diff
@@ -158,16 +214,21 @@ const JurisdictionSelect = (props: JurisdictionSelectProps & FieldProps) => {
       setHierarchy([]);
       setShouldMenuOpen(false);
       setCloseMenuOnSelect(false);
+      setLowestLocation(false);
+      setIsJurisdiction(true);
       // set the Formik field value
       if (form && field) {
         form.setFieldValue(field.name, '');
       }
     }
   };
-
+  // const clearSelect = () => {
+  //   refs.async.select.state.value = '';
+  // };
```

Comment:
Why is this commented out?

Reply:
Was this meant to implement the reset functionality?

Reply:
An attempt but it didn't work

Reply:
I don't think that it is a good idea to have hanging commented out code.  Better to just remove it.